package/gluon-status-page-api/src/stations.c

```diff
@@ -58,8 +62,46 @@ bool interface_is_valid(const char *ifname) {
   return access(path, F_OK) == 0;
 }
 
+int interface_get_lowest(const char *ifname, char *hwifname);
+int interface_get_lowest(const char *ifname, char *hwifname) {
```

Comment:
Every time I look at the code, I see more issues...

interface_get_lowest() should be static, and there's no reason to add a prototype directly above the definition.